server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -59,16 +61,22 @@ public class TranslogTransferManager {
     private final static String METADATA_DIR = "metadata";
 
     public TranslogTransferManager(
+        ShardId shardId,
         TransferService transferService,
         BlobPath remoteBaseTransferPath,
         FileTransferTracker fileTransferTracker
     ) {
+        this.shardId = shardId;
         this.transferService = transferService;
         this.remoteBaseTransferPath = remoteBaseTransferPath;
         this.remoteMetadataTransferPath = remoteBaseTransferPath.add(METADATA_DIR);
         this.fileTransferTracker = fileTransferTracker;
     }
 
+    public ShardId getShardId() {
```

Comment:
lets remove this?

Reply:
Added `shardId` to critical deletion logs .

Reply:
Cool, that's good. Should we keep the log level to INFO or change it to trace for deletion as it can be frequent,